src/Login.cpp

```diff
@@ -141,4 +137,25 @@ std::shared_ptr<User> Login::verifyUser(const std::string name, const std::strin
     }
 
     return NULL;
+}
+
+bool Login::changePermission(std::string username, std::string account, int permission) {
+    int p;
+
+    if (account == "employee") p = 1;
+    if (account == "manager") p = 3;
+    if (account == "owner") p = 5;
+
```

Comment:
I think there needs to be some sort of check to verify that the account type you've passed in is one of the support types.

It looks like with this, I'm able to pass in anything for account type, and as long as I have high enough privilege, I can do.

I was able to login to an admin account and change another account to account type ```faketype```, which doesn't exist.

So, my proposal is:

```
if (account == "employee") {
    p = 1;
} else if (account == "manager") {
    p = 3;
} else if (account == "owner") {
    p = 5;
} else {
    Logger::logError("No such account type '%s'.", account.c_str());
    return false;
}
```